i18n/en.json

```diff
@@ -3,26 +3,29 @@
   "info": "Help translate text labels on SVG files. Pick a file on commons to get started.",
   "beta": "This feature is in early testing.",
   "learn-more": "Learn more & leave feedback!",
-  "powered-by-toolforge": "Powered by Toolforge",
-  "report-a-bug": "Report a bug",
-  "privacy-policy": "Privacy policy",
-  "view-on-github": "View on Github",
-  "licenced-under": "Licenced under $1",
-  "gpl3": "GPL 3.0 or later",
-  "made-by": "Made by the Wikimedia Foundation's $1",
-  "commtech": "Community Tech team",
+
   "help": "Help",
-  "pick-an-image": "Pick an image",
-  "pick-an-image-more": "from Commons.",
-  "translate-labels": "Translate",
-  "translate-labels-more": "the labels in the image.",
-  "update-commons": "Update Commons",
-  "update-commons-more": "or download directly to your computer.",
   "login": "Log in",
   "logout": "Log out",
   "language-settings": "Language settings",
+
+  "search-help": "Type a file name from Commons",
   "search-label": "File name (without 'File:' prefix) or URL:",
   "search-placeholder": "100 Years War France 1435.svg",
   "translate-button": "Translate",
-  "search-help": "Type a file name from Commons"
+
+  "pick-an-image": "<strong>Pick an image</strong> <span>from Commons.</span>",
+  "translate-labels": "<strong>Translate</strong> <span>the labels in the image.</span>",
+  "update-commons": "<strong>Update Commons</strong> <span>or download directly to your computer.</span>",
```

Comment:
Are the spans necessary? Just leaves more room for error, say if they misspelled it or forgot the closing tag

Reply:
This is actually bigger of an issue (I think you're talking about this elsewhere in the review, @MusikAnimal ) -- this will be "raw", and that's a pretty big security risk especially since the i18n bot self-merges into the code.

We should avoid 'raw' anywhere we can. I flagged this specific one for @Niharika29 to see if the design in this case can be flexible to not have the partial-bold part, but we should keep in mind to try and avoid html messages we display as raw and render into the DOM as much as possibly possible.

Reply:
I think for these messages it's better to split the strings up into two parts such as `tutorial-image-head` and `tutorial-image-desc', `tutorial-translate-head` and `tutorial-translate-desc` etc. For the ones below, `licensed-under` and `made-by`, I agree with Leon that we can make them complete links and also we can switch the wording to what we use in XTools. 
Let's also document these standard wordings we use in our tools somewhere on MW so we don't have to spend time thinking about them every time and we can save @prtksxna the bother. :)

Reply:
I had these in two parts in the previous iteration of this patch. I thought it was bad to split single phrases up for translation, so combined them. I guess two messages is fine if the qqq docs are clear about how the messages are going to be displayed?

The spans are necessary because the two parts of these phrases are displayed differently, and both as blocks. But I'll switch back to multiple messages, so that's moot.

Reply:
Yeah, I agree with everything you say there. Done.